pkgs/development/python-modules/Theano/default.nix

```
@@ -81,7 +77,8 @@ in buildPythonPackage rec {
 
   # keep Nose around since running the tests by hand is possible from Python or bash
   checkInputs = [ nose ];
-  propagatedBuildInputs = [ numpy numpy.blas scipy six libgpuarray_ ];
+  # setuptools needed for cuda support
+  propagatedBuildInputs = [ numpy numpy.blas scipy setuptools six libgpuarray_ ];
```

this is getting a bit long
```suggestion
  propagatedBuildInputs = [
    numpy 
    ...
  ];
```